tests.py

```diff
@@ -70,6 +96,80 @@ def testParseArguments(self):
     de_object.ParseOptions(options)
     self.assertEqual(expected_docker_root, options.docker_directory)
 
+  def testShowHistory(self):
+    """Tests that ShowHistory shows history."""
+    de_object = de.DockerExplorer()
+    # We pick one of the container IDs.
+    container_id = container.GetAllContainersIDs(self.docker_directory_path)[0]
+    with mock.patch('sys.stdout', new=StringIO()) as fake_output:
+      de_object.docker_directory = self.docker_directory_path
+      de_object.ShowHistory(container_id)
+      expected_dict = {
+          ('sha256:'
+           '8ac48589692a53a9b8c2d1ceaa6b402665aa7fe667ba51ccc03002300856d8c7'):
+          {
+              'container_cmd': '/bin/sh -c #(nop)  CMD [\"sh\"]',
+              'created_at': '2018-04-05T10:41:28.876407', 'size': 0}
+      }
+      self.assertEqual(
+          utils.PrettyPrintJSON(expected_dict)+'\n', fake_output.getvalue())
```

Comment:
Maybe this string could be hardcoded somewhere instead of calling `PrettyPrintJSON`?

Reply:
done. Will make hardcoded strings homogeneous in a future PR